ppp/ppp_test.go

```diff
@@ -1,3 +1,4 @@
+// package ppp implements establishing a PPP connection running over PPPoE.
```

Comment:
Capital Package. And s/implements establishing/establishes/.

Reply:
Done.